### PR TITLE
role-phoebus: fix version handling in installation.

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: tasks-desktops.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
+- import_role:
+    name: lnls-ans-role-repositories
```
Comment (lerwys) [inline tasks-desktops.yml]: Sorry for being picky. =] But doesn't this change belong in another PR?

Comment (xresende) [reply tasks-desktops.yml]: you are right. "given enough eyeballs all bugs and lazy commits are shallow", I heard. done.
